jsonnet/components/grafana.libsonnet

```diff
@@ -17,25 +17,6 @@ function(params)
 
   grafana(cfg) {
 
-    consoleDashboardDefinitions: {
```

Comment:
we don't want to drop all dashboards but only the etcd one. Instead of `std.map()` we need [`std.filterMap()`](https://jsonnet.org/ref/stdlib.html#filterMap).

Reply:
Indeed! Should be good now